### PR TITLE
Refactor Multiownable

### PR DESCRIPTION
**Pull request by wilsoncusack**
Originally I used unit8 to track owner index because I was packing this value. Later I switched to use SignatureWrapper and be more explicit, and rely on rollup compression to give me the `0`s for free. 

As this is now the case, we can simplify MultiOwnable to use `uint256` for index, which allows us to remove `addOwnerAtIndex` at simplify the code.

### CODE REVIEW

File: src/MultiOwnable.sol [hunk 1]
```diff
@@ -3,16 +3,19 @@ pragma solidity ^0.8.4;
...
-    /// @dev Allows an owner to be idenfitied by a uint8.
-    /// Seco256r1 verifier does not recover the address, but requires
-    /// the X,Y coordinates to be passed for verification.
-    /// In the context of checking whether something was signed by an owner
-    /// this means that the signature needs to include an identifier of the owner.
-    /// In an effort to economize calldata, we use a uint8 rather than passing the
-    /// X,Y coordinates.
-    mapping(uint8 => bytes) ownerAtIndex;
+    /// @dev tracks the index of the next owner added
+    uint256 nextOwnerIndex;
```
Comment (pegahcarter) [inline src/MultiOwnable.sol:7]: I'm opinionated on still using owner indices.

Comment (wilsoncusack) [reply src/MultiOwnable.sol:7]: wdym?

File: .gitmodules [hunk 2]
```diff
@@ -16,3 +16,4 @@
 [submodule "lib/webauthn-sol"]
 	path = lib/webauthn-sol
 	url = https://github.com/base-org/webauthn-sol
+	branch = wilson/fix
```
Comment (anikaraghu) [inline .gitmodules:19]: is there some change that still needs to be merged upstream?

Comment (wilsoncusack) [reply .gitmodules:19]: oops, need to clean up

File: src/MultiOwnable.sol (hunk 1 above)
Comment (pegahcarter) [reply src/MultiOwnable.sol:7]: I would use something like [EnumerableSet](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/structs/EnumerableSet.sol) to maintain the owners array without empty indices.

Comment (wilsoncusack) [reply src/MultiOwnable.sol:7]: Why?

Comment (wilsoncusack) [reply src/MultiOwnable.sol:7]: Ah I see your edit, fair comment! Don't want to take on that scope right now as I think this meets our requirements with no downsides. Will keep in mind.

**Pull request merged**